Mastodon/Scene/Share/View/Content/PollOptionView+Configuration.swift

```diff
@@ -109,3 +114,30 @@ extension PollOptionView {
             .store(in: &disposeBag)
     }
 }
+
+extension PollOptionView {
+    public func configure(historyPollOption option: StatusEdit.Poll.Option) {
+        // background
+        ThemeService.shared.currentTheme
+            .receive(on: DispatchQueue.main)
+            .sink { [weak self] theme in
+                guard let self = self else { return }
+                self.viewModel.roundedBackgroundViewColor = theme.systemElevatedBackgroundColor
+            }
+            .store(in: &disposeBag)
+        // metaContent
+        viewModel.metaContent = PlaintextMetaContent(string: option.title)
+        // show left-hand-side dots, otherwise view looks "incomplete"
+        viewModel.selectState = .off
+        // appearance
+        ThemeService.shared.currentTheme
+            .receive(on: DispatchQueue.main)
+            .sink { [weak self] theme in
+                guard let self = self else { return }
+                self.checkmarkBackgroundView.backgroundColor = UIColor(dynamicProvider: { trailtCollection in
+                    return trailtCollection.userInterfaceStyle == .light ? .white : theme.tableViewCellSelectionBackgroundColor
```

Comment:
Is there a reason why we use `.white` for `.light` instead of relying on the asset-catalog-color?

Reply:
Good question this code has been in place for a while and I'm not able to reason about it, would need to further investigate this.